src/YouTubeSubtitlesExtractor/Models/VideoDetail.cs

```diff
@@ -0,0 +1,22 @@
+namespace Aliencube.YouTubeSubtitlesExtractor.Models;
+
+/// <summary>
+/// This represents the video details entity.
+/// </summary>
+public class VideoDetails
+{
+    /// <summary>
+    /// Gets or sets the video title.
+    /// </summary>
+    public string Title { get; set; }
+
+    /// <summary>
+    /// Gets or sets the video description.
+    /// </summary>
+    public string Description { get; set; }
+
+    /// <summary>
+    /// Gets or sets the list of available subtitle languages.
+    /// </summary>
+    public List<string> AvailableSubtitleLanguages { get; set; }
```

Comment:
For now, we haven't decided which properties we're going to use. Would you please remove them for now?